README.md

```diff
@@ -16,3 +16,7 @@ Install the `libvips` library. On Mac OS you can run `brew install libvips`.
 
 Run `jekyll serve -l` to run jekyll with live reload. You can access the site at
 <a href="http://localhost:4000".
+
+## Employee images
+
+When you add a new employee, make sure to add a new image to the `assets/images/employees` folder. The image should be 400x400 pixels. The filename should be the same as the employee's name, but with dashes instead of spaces. For example: `jane-doe.jpg`. and then run `ruby modify_team_data.rb` to update the teammembers.yml file.
```

Comment:
Wenn möglich lass uns größere Bilder verwenden – so groß wie möglich. `jekyll_picture_tag` sollte sich dann drum kümmern, die für verschiedene Devices in die richtige Größe zu bringen.